authentikos/README.md

```diff
@@ -37,9 +37,11 @@ The following is a list of supported options for `authentikos`:
 
 ```console
   -c, --creds string           Path to a JSON credentials file.
+  -r, --force-refresh          Force a token refresh. Otherwise, the token will only refresh when necessary.
+  -i, --interval int           Token refresh interval in minutes [1 - 60). (default 30)
```

Comment:
nit: Consider using a duration string instead. e.g. `2m`.

Reply:
That would be a nice improvement. I will update!

Reply:
Updated. Added unit tests to support this change as well.